components/KnitHeader.vue

```diff
@@ -86,21 +93,22 @@
 
   &__link {
     display: flex;
+    align-items: center;
     text-decoration: none;
     color: $knit-header-text-color;
   }
 
   &__logo {
-    width: 35px;
+    width: 45px;
```

Comment:
It doesn't look good. It's too big. Please use max 37px width.

Reply:
I left a few comments about prefetching methods, interesting topic :)

Reply:
I disagree it looks much better @kwojcik please take a stand

Reply:
Especially on monitors with low dpp